### DISCUSSION TITLE
India 🇮🇳

### DISCUSSION THREAD

**Discussion by uby777881** [Semporia/TikTok-Unlock#317] [General]
您好，我尝试了这些步骤，但在印度不起作用。如果我想从印度使用它，是否需要进行任何更改？

我注意到印度也禁止了tiktok，无法使用SIM cadd访问该应用程序。如果我想进行任何更改，请告诉我。 谢谢你

**Comment by Semporia** [comment]
如插入SIM卡按照教程配置即可！

**Reply by uby777881** [reply]
我已经尝试过但没有成功。我需要更改 URL 重写中的任何内容吗
而不是CN，我需要换成IN？因为目前我在印度，我想从印度访问
(?<=_region=)**CN**

(?<=_region=)CN(?=&) url 307 KR
(?<=&mcc_mnc=)4 url 307 2
^(https?:\/\/(tnc|dm)[\w-]+\.\w+\.com\/.+)(\?)(.+) url 302  $1$3
(?<=\d\/\?\w{7}_\w{4}=)1[6-9]..(?=.?.?&) url 307 17

**Reply by Semporia** [reply]
(?<=_region=)IN(?=&) url 307 US
(?<=&mcc_mnc=)460[0-1][0-9](?=&) url 307 310005
^(https?:\/\/(tnc|dm)[\w-]+\.\w+\.com\/.+)(\?)(.+) url 302  $1$3
(?<=&carrier=)[%A-Z0-9]+(?=&) url 307 Verizon
(?<=tz_name=)Asia/India(?=&) url 307 America/New_York
(?<=timezone=)8(?=&) url 307 -4
